include/revng/Model/Binary.h

```diff
@@ -108,6 +108,14 @@ class model::Binary : public model::generated::Binary {
                                 "/Types/" + getNameFromYAMLScalar(T->key()));
   }
 
+  model::TypePath getTypePath(const model::Type::Key &Key) {
+    return TypePath::fromString(this, "/Types/" + getNameFromYAMLScalar(Key));
+  }
+
+  model::TypePath getTypePath(const model::Type::Key &Key) const {
+    return TypePath::fromString(this, "/Types/" + getNameFromYAMLScalar(Key));
+  }
```

Comment:
The other two overloads should call these two methods.

Reply:
Done: https://github.com/revng/revng/pull/197/commits/b31bc794623792d25b85584abd2a301e2657a550#diff-612e49a68d142703a9b058750b7936a1e80e98624734600fd4b22692aab9338bR110